yoti_python_sdk/image.py

```diff
@@ -0,0 +1,32 @@
+from yoti_python_sdk.protobuf.protobuf import Protobuf
+
+
+class Image:
+    def __init__(self, image_bytes, image_content_type):
+        if image_content_type == Protobuf.CT_JPEG or image_content_type == Protobuf.CT_PNG:
```

Comment:
Could we use a dictionary of allowed image types and their mime type to reduce `if/elif` and `or`? e.g.
```python
self.__image_types = {
    Protobuf.CT_JPEG: 'image/jpeg',
    Protobuf.CT_PNG: 'image/png',
}
```

Then you could do something like:
```python
    def __init__(self, image_bytes, image_content_type):
         if image_content_type in self.__image_types:
         ...
```

```python
    def mime_type(self):
        if self.__content_type in self.__image_types:
            return self.__image_types[self.__content_type]
        else:
            ...
```

Reply:
I like the idea of extracting the "allowed_types", so I will do that. Not sure if it's worth adding the dictionary to include the mime type, as we only use that in one method currently.